### PR TITLE
Start of versioning page. Covers basics only.

### PR DESCRIPTION
**Pull request by odscjames**
Details to be fleshed out with more Loomio discussion.

### CODE REVIEW

File: versioning.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Versioning our libraries
+========================
+
+Which repositories?
+-------------------
+
+Some of our repositories are used as libraries and included as a requirement in other repositories.
+eg https://github.com/OpenDataServices/flatten-tool/releases
+
+For these libraries, we want to version them to make it easier to do this.
```
Comment (robredpath) [inline versioning.md]: I think that docs should state what we do, not what we want to do. So, we should say "For these libraries, we version them to make it easier to do this. This applies to all new libraries. As of July 2018, work to version existing libraries is still in progress".

File: versioning.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Creating versions, and the branching scheme
+-------------------------------------------
+
+Versions are created as tags in GitHub, with the format "vX.Y.Z". Semantic versioning should be followed. https://semver.org/
+
+Work is merged to the "master" branch, and versions are tagged directly on the master branch.
+
+This means that when a later version is released, no bug fixes to earlier versions will be released any more.
```
Comment (robredpath) [inline versioning.md]: +1

File: versioning.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Versioning our libraries
+========================
+
+Which repositories?
+-------------------
+
+Some of our repositories are used as libraries and included as a requirement in other repositories.
+eg https://github.com/OpenDataServices/flatten-tool/releases
```
Comment (robredpath) [inline versioning.md]: Is the /releases page the correct one to link to here?

**Comment by robredpath** [comment]
This seems sensible. We should do the work on this in response to identified user need - eg flatten-tool has been requested, I believe?

File: versioning.md (hunk 3 above)
Comment (odscjames) [reply versioning.md]: Well, it's just an example link, and given the whole point of this page is released versions then releases seems like a good page to link to.

File: versioning.md (hunk 1 above)
Comment (odscjames) [reply versioning.md]: I'm not sure we have fully agreed about this - and I note your later comment talks about User needs, which might contradict this. Can we leave this pull request at it is (just saying "some", which is technically true) and have some Loomio discussion to clarify and make more specific points like this later?

**Pull request merged**